.travis.yml

```diff
@@ -10,12 +10,12 @@ notifications:
   email: false
 
 jobs:
-  include:
+  include:  
```

Comment:
prettier can lint yaml too in latest release :laughing:

Reply:
Adding it now!